src/components/ui/LoginForm.tsx

```diff
@@ -1,45 +1,17 @@
 'use client';
-import { Amplify } from 'aws-amplify';
-import { fetchAuthSession, signInWithRedirect } from 'aws-amplify/auth';
+import { getToken } from '@/lib/api/authTokenFetcher.client';
```

Comment:
The reason why I import directly from `@/lib/api/authTokenFetcher.client`, not from `@/lib/api` is to throw `ReactServerComponentsError`.

Reply:
This problem is solved with this [commit](https://github.com/genesis-tech-tribe/nishiki-frontend/pull/187/commits/649b3d50af43cc0f612f9c4a06d6d5b9c680fd8a) for over all directory refactoring for api client files.